lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -1,6 +1,13 @@
 module ShopifyAPI
   class FulfillmentOrder < Base
+    def self.version_validation!
+      available_in_version = ShopifyAPI::ApiVersion.find_version('2020-01')
+      raise NotImplementedError unless ShopifyAPI::Base.api_version >= available_in_version
```

Comment:
Would be good to also include a message indicating the unsupported version or minimum required version.